### PR TITLE
[Design guidance] Additional pages

### PR DESCRIPTION
**Pull request by niels9001**
## Description
- Updated the design guidance menu names to be inline with the docs (e.g. Icons => Iconography, Colors => Color) and sorted alphabetically.
- Moved the Card stroke brush on the Stroke colors page to the top, so it's inline with the Card background brush and the control strokes are grouped.
- Adding a bit more spacing between the color sections
- Left aligned type ramp image on the Typography page
- Added a new Geometry page explaining how to use the CornerRadius resources:
image

- Added a new Spacing page showing a spacing ramp and some common UI examples with redlines:
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                    <Setter Property="Grid.Column" Value="2" />
+                    <Setter Property="TextWrapping" Value="Wrap" />
+                    <Setter Property="VerticalAlignment" Value="Center" />
+                </Style.Setters>
+            </Style>
+
+            <Style x:Key="VisualBorderStyle" TargetType="Border">
+                <Style.Setters>
+                    <Setter Property="Grid.Column" Value="1" />
+                    <Setter Property="CornerRadius" Value="{StaticResource ControlCornerRadius}" />
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:66]: If it were me, i wouldn't round corners here.

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <RichTextBlock>
+            <Paragraph>
+                <Run>The use of consistently sized spacing and gutters semantically groups an experience into separate components. These values map to our rounded corner logic and together help create a cohesive and usable layout.</Run>
+                <Run>A best practice in design is to use a 4px grid. This means that any spacing or sizing should be a multiple of 4. This helps to create a consistent and harmonious layout.</Run>
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:84]: ... And makes for nice values when scaling

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+                    </Grid>
+                    <!--  48epx  -->
+                    <Grid
+                        Background="{ThemeResource CardBackgroundFillColorDefaultBrush}"
+                        Grid.ColumnDefinitions="86,136,*"
+                        Style="{StaticResource GridCardStyle}">
+                        <TextBlock Style="{StaticResource ValueTextStyle}" Text="48epx" />
+                        <Border Width="48" Style="{StaticResource VisualBorderStyle}" />
+                        <TextBlock Style="{StaticResource DescriptionTextStyle}" Text="Spacing between page sections with title." />
+                    </Grid>
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:191]: Why not make all this in one grid?

**Review by Jay-o-Way — COMMENTED** [review]
Nice! Few small comments.

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml (hunk 1 above)
Comment (niels9001) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:66]: We use this on the typo table as well

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml (hunk 2 above)
Comment (niels9001) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:84]: Added

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml (hunk 1 above)
Comment (Jay-o-Way) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:66]: These colored areas are not controls and are not used on Typo page :)

**Comment by Jay-o-Way** [comment]
Detail inconsistent
image

**Comment by Jay-o-Way** [comment]
Just thinking: maybe F0B4 "Protractor" could also be a nice icon for the Geo page?

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
@niels9001 Detail inconsistent
image

**Comment by niels9001** [comment]
> @niels9001 Detail inconsistent image

Aah, great catch! Updated the image!

**Comment by Jay-o-Way** [comment]
A thought about the icon?

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
Still inconsistent (internally)
image

_and_ inconsistent with the linked page on MS Learn
image

CC @crutkas

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml (hunk 1 above)
Comment (Jay-o-Way) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:66]: @niels this is what I mean:
image

**Review by Jay-o-Way — CHANGES_REQUESTED** [review]
By the definition seen on the Geometry page, every instance of `<Grid CornerRadius="{StaticResource OverlayCornerRadius}"...>` - which are the light-colored items in a "fake list" - should actually be using **ControlCornerRadius**
image

Speaking of those "lists" - there **has** to be an easier control, right?

File: WinUIGallery/ControlPages/DesignGuidance/GeometryPage.xaml [hunk 4]
```diff
@@ -0,0 +1,273 @@
...
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="148" />
+                                <ColumnDefinition Width="*" MaxWidth="400" />
+                                <ColumnDefinition Width="112" />
+                            </Grid.ColumnDefinitions>
+                            <TextBlock
+                                Margin="16,0,0,0"
+                                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+                                Style="{StaticResource CaptionTextBlockStyle}"
+                                Text="Corner radius" />
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/GeometryPage.xaml:138]:
```suggestion
                        <Grid Margin="16,48,0,24" HorizontalAlignment="Stretch">
                            <Grid.ColumnDefinitions>
                                <ColumnDefinition Width="148" />
                                <ColumnDefinition Width="*" MaxWidth="400" />
                                <ColumnDefinition Width="112" />
                            </Grid.ColumnDefinitions>
                            <TextBlock
                                Foreground="{ThemeResource TextFillColorSecondaryBrush}"
                                Style="{StaticResource CaptionTextBlockStyle}"
                                Text="Corner radius" />
```
and so on

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+                    <Setter Property="Grid.Column" Value="2" />
+                    <Setter Property="TextWrapping" Value="Wrap" />
+                    <Setter Property="VerticalAlignment" Value="Center" />
+                </Style.Setters>
+            </Style>
+
+            <Style x:Key="VisualBorderStyle" TargetType="Border">
+                <Style.Setters>
+                    <Setter Property="Grid.Column" Value="1" />
+                    <Setter Property="CornerRadius" Value="{StaticResource ControlCornerRadius}" />
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:66]:
```suggestion
```

File: WinUIGallery/ControlPages/DesignGuidance/GeometryPage.xaml [hunk 6]
```diff
@@ -0,0 +1,273 @@
...
+                            <StackPanel Orientation="Horizontal" Spacing="12">
+                                <Border
+                                    Grid.Column="1"
+                                    Width="20"
+                                    Height="20"
+                                    Margin="16,0,0,0"
+                                    Background="{ThemeResource AccentFillColorDefaultBrush}"
+                                    CornerRadius="{StaticResource OverlayCornerRadius}" />
+                                <TextBlock VerticalAlignment="Center" Text="8px" />
+                            </StackPanel>
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/GeometryPage.xaml:170]: I would just use an extra column in the Grid for this. The StackPanel is kinda sticking out here.

**Comment by Jay-o-Way** [comment]
Looks like this isn't pixel-perfect either
image

**Comment by Jay-o-Way** [comment]
**Padding** of **24** in the "lists" is too large; ListView uses **12**
image

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml [hunk 7]
```diff
@@ -121,13 +129,12 @@
             Style="{StaticResource GalleryTileGridStyle}">
             <ScrollViewer HorizontalScrollBarVisibility="Auto" HorizontalScrollMode="Auto">
                 <StackPanel Padding="12,24,12,12">
-                    <Grid HorizontalAlignment="Stretch">
+                    <Grid Margin="16,0,0,0" HorizontalAlignment="Stretch">
```
Comment (Jay-o-Way) [inline WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:132]: Moving the margin was a bad change on my end. #oops

Comment (niels9001) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:124]: Fixed it

**Comment by niels9001** [comment]
@jay-o-way Made some tweaks based on your feedback, I think for the spacing on the lists for spacing / typeramp we are good as its per what the design folks created for us

**Review by karkarl — APPROVED** [review]

File: WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml (hunk 5 above)
Comment (niels9001) [reply WinUIGallery/ControlPages/DesignGuidance/SpacingPage.xaml:57]: I think adding a bit of rounding makes it feel more inline with all the other UI elements

**Pull request merged**